client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -497,28 +600,13 @@ const TestPlanVersionsPage = () => {
                     })}
                 </tbody>
             </ThemeTable>
-
-            {testPlanVersions.map(testPlanVersion => {
-                const vString = `V${convertDateToString(
-                    testPlanVersion.updatedAt,
-                    'YY.MM.DD'
-                )}`;
-
-                // Gets the derived phase even if deprecated by checking
-                // the known dates on the testPlanVersion object
-                const derivedDeprecatedAtPhase =
-                    deriveDeprecatedDuringPhase(testPlanVersion);
-
-                const hasFinalReports =
-                    (derivedDeprecatedAtPhase === 'CANDIDATE' ||
-                        derivedDeprecatedAtPhase === 'RECOMMENDED') &&
-                    !!testPlanVersion.testPlanReports.filter(
-                        report => report.isFinal
-                    ).length;
-
-                return (
-                    <div key={testPlanVersion.id}>
-                        <H2
+            <PageSpacer />
+            <DisclosureComponent
+                componentId="versionHistory"
+                title={testPlanVersions.map(testPlanVersion => {
```

Comment:
The console throws the following errors:

```
Warning: Failed prop type: Invalid prop `title` supplied to `DisclosureComponent`, expected one of type [string, string].
```

and 

```
Warning: Encountered two children with the same key, `disclosure-[object Object]-key`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

And that's because the DisclosureComponent has been accepting plain string as title values in the array and the constructed key is also based on that string but here, the title array is being passed as `[ <React.Element>, ... ]` so you may want to:

1. Relax the constraints on the title props being accepted by the DisclosureComponent, and
2. Allow for explicitly key values to be used, to be set